### DISCUSSION TITLE
Quienes son explorers de Perú? 🇵🇪

### DISCUSSION THREAD

**Discussion by VictorFalconi** [Launch-X-Latam/MisionFrontEnd#90] [General]
Hola! Mi nombre es Victor. Soy de Lima, Perú. Quería saber quienes son los explorers de Perú para poder ayudarnos entre nosotros. Sobre todo para hacer networking ya que trabajaremos en el mismo entorno. Si son de Perú y quieren colaborar, respondan en la discusion. Dejo mi LinkedIn por si quieren conectar https://www.linkedin.com/in/victor-jose-falconi-medina

**Comment by maurpz** [comment]
Hola compa :v tmb soy de peru

**Reply by VictorFalconi** [reply]
Hola, pasa tu LinkedIn para conectar 🙌🏽

**Comment by Tobsad** [comment]
x2 yo tambien soy de Perukistan xd

**Reply by VictorFalconi** [reply]
Hola, pasa tu LinkedIn para conectar 🙌🏽

**Comment by renzocastro** [comment]
Hola, aquí otro de Lima =)

**Reply by VictorFalconi** [reply]
Hola, pasa tu LinkedIn para conectar 🙌🏽

**Comment by DanteAllccahuaman** [comment]
Hola, aquí también de Perú.

**Reply by VictorFalconi** [reply]
Hola, pasa tu LinkedIn para conectar 🙌🏽

**Comment by YovaniHuarsaya** [comment]
Perú tmbn 😅

**Reply by VictorFalconi** [reply]
Hola, pasa tu LinkedIn para conectar 🙌🏽

**Comment by gersonjeanpierre** [comment]
Aqui de lima

**Reply by VictorFalconi** [reply]
Hola, pasa tu LinkedIn para conectar 🙌🏽

**Comment by jonathanbarzola** [comment]
¡Hola! Yo también soy de Lima, Perú. aquí dejo mi [Linkedin](https://www.linkedin.com/in/jonathanbarzola/)
